### ISSUE TITLE
NullPointerException when saving new card after rotating the device

### ISSUE THREAD

**Issue by cddf** [stefan-niedermann/nextcloud-deck#270]
**Describe the bug**
The app sometimes crashes when a new card is saved.
It's not completely clear for me, how to reproduce it always.

**To Reproduce**
1. Make a clean start of the app
2. Rotate 90°
3. Click the + to create a new card
    - When a card has already been saved without rotating it works without crash
4. Save the card
5. App crashes

**Expected behavior**
It should not crash, but save the card.

**Smartphone (please complete the following information):**
 - Device: LG G5
 - Android-Version: 8

<details>
<summary>
<b>Stacktrace</b>
</summary>

App Version: 0.0.17
App Version Code: 17
Files App Version Code: 30100090

---

OS Version: 3.18.71-perf+(1832611538957)
OS API Level: 26
Device: h1
Model (and Product): LG-H850 (h1_global_com)

---
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Long it.niedermann.nextcloud.deck.model.full.FullStack.getLocalId()' on a null object reference
	at it.niedermann.nextcloud.deck.persistence.sync.SyncManager.lambda$createFullCard$17$SyncManager(SyncManager.java:502)
	at it.niedermann.nextcloud.deck.persistence.sync.-$$Lambda$SyncManager$7UomFrsDF0d9oANwwzsN12JJGYM.run(Unknown Source:12)
	at java.lang.Thread.run(Thread.java:764)
```

</details>

**Comment by desperateCoder** [comment]
@cddf thanks for your report! As I'm the "Server-Synchronisation" guy I never tried that tbh, but this should definitely not happen (no s**t Sherlock!)

We'll have a look at this, maybe on Saturday. 

@stefan-niedermann no idea what Android does when rotating, but maybe it is similar to the crah-after-nightmode-toggle thingy, just to have that in mind. Could be a timing problem, let's test the steps described by @cddf really fast.

**Comment by stefan-niedermann** [comment]
This should be fixed with `0.7.0` (not yet released), please reopen if this issue appears again.

Thank you for reporting 🙂
